src/Sarif/Readers/FlagsEnumConverter.cs

```diff
@@ -53,7 +53,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 throw new ArgumentNullException(nameof(writer));
             }
 
-            if (!(value is ArtifactRoles))
+            if (!(value is ArtifactRoles || value is ToolComponentContents))
```

Comment:
>if (!(value is ArtifactRoles  [](start = 12, length = 29)

Add a comment here please. Open an issue to detect whether the enum type is decorated with flags and disregard in this case.

Reply:
Actually, remove this entirely

---
In reply to: [269752447](https://github.com/Microsoft/sarif-sdk/pull/1373#discussion_r269752447) [](ancestors = 269752447)